pageserver/src/tenant/storage_layer.rs

```diff
@@ -104,10 +121,10 @@ struct LayerAccessStatsInner {
 }
 
 #[derive(Debug, Clone)]
```

Comment:
Might be worthwhile to add `#[derive(Copy)]` to these which are always memcpyable.

Reply:
https://github.com/neondatabase/neon/pull/3574